src/2_add_cots_priority.jl

```diff
@@ -39,5 +42,8 @@ rename!(RRAP_lookup, Dict(:priority=>:COTS_priority))
 RRAP_lookup.COTS_priority .= ifelse.(ismissing.(RRAP_lookup.COTS_priority), "NA", RRAP_lookup.COTS_priority)
 RRAP_lookup.COTS_priority = convert.(String, RRAP_lookup.COTS_priority)
 
+# Revert incorrect GBRMPA_ID to match data provided by GBRMPA
+RRAP_lookup.GBRMPA_ID = ifelse.(RRAP_lookup.GBRMPA_ID .== "20-198", "20198", RRAP_lookup.GBRMPA_ID)
+
```

Comment:
Why do this? Why not delete Lines 8-10?

Shouldn't  `GBRMPA_ID` come from `gbr_features`? (it's `LABEL_ID` I believe)

Reply:
The COTS data contains the fixed ID ("20-198") so to join the COTS information to the dataset it needs to be temporarily fixed

Reply:
Okay, sounds like a pragmatic workaround for now. We'll clean up later I guess.